File: C_Sharp/C_Sharp/domain/Reservation.cs
namespace C_Sharp.domain
{
    public class Reservation : Entity<long>
    {
        public string Name { get; set; }
        public long PhoneNumber { get; set; }
        public long TicketsNumber { get; set; }
        public long ExcursionId { get; set; }

        public Reservation(string name, long phoneNumber, long ticketsNumber, long excursionId)
        {
            Name = name;
            PhoneNumber = phoneNumber;
            TicketsNumber = ticketsNumber;
            ExcursionId = excursionId;
        }
    }
}

File: MPP-Java/Services/src/main/java/domain/Reservation.java
package domain;

public class Reservation extends Entity<Long>{
    private String name;
    private Long phoneNumber;
    private Long ticketsNumber;
    private Long excursionID;

    public Reservation(String name, Long phoneNumber, Long ticketsNumber, Long excursionID) {
        this.name = name;
        this.phoneNumber = phoneNumber;
        this.ticketsNumber = ticketsNumber;
        this.excursionID = excursionID;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(Long phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public Long getTicketsNumber() {
        return ticketsNumber;
    }


    public void setTicketsNumber(Long ticketsNumber) {
        this.ticketsNumber = ticketsNumber;
    }

    public Long getExcursionID() {
        return excursionID;
    }

    public void setExcursionID(Long excursionID) {
        this.excursionID = excursionID;
    }

    @Override
    public String toString() {
        return "Reservation{" +
                "name='" + name + '\'' +
                ", phoneNumber=" + phoneNumber +
                ", ticketsNumber=" + ticketsNumber +
                ", excursionID=" + excursionID +
                '}';
    }


}
